### ISSUE TITLE
Update front-end to use more recent version of turf library

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#1092]
[Turf](http://turfjs.org/) is a geospatial analysis Javascript library that we use extensively on the front-end. We are currently using a rather old version of the library, and a lot of new features/functions have been added in more recent releases.

I needed a more recent version of this library to create the IRR calculation tool ( #1015 ) and the accuracy calculation tool ( #1059 ). However, the newer version of the library breaks the tool, so I can't merge that code upstream. It would be nice to use a more recent version of the library in our production code so that we have the ability to use these newer features if we ever want to, and also so that I can merge the code I have written upstream.

**Comment by jonfroehlich** [comment]
After CHI, can you fill in the gaps on what seems to be breaking?

**Comment by misaugstad** [comment]
I could, yes, though it would probably be a lot faster for someone who has more experience with the front end to do it (and someone who likes working with the front-end more :wink: ). But if no one else steps forward I can, of course, go through and do this.

**Comment by misaugstad** [comment]
@paarig how do you feel about picking up this as your next task?

**Comment by paarig** [comment]
@misaugstad Sounds like a plan. The current task I'm working on has thrown me for a loop so it might be a little bit before I can get started on this task.

**Comment by misaugstad** [comment]
No worries! Also feel free to take a break from your current task to work on this if you are feeling stuck. Whatever works for you :)

**Comment by paarig** [comment]
@misaugstad Could you walk me through what changes to merge from the old feature branches you worked in? I upped the versioning of turf but it was tough for me to merge any changes from those branches since I don't know which of them are changes that are part of the calculation tool and which are part of changes made in previous develop commits that have since been overwritten by more recent changes.

**Comment by misaugstad** [comment]
okay I see, I should have updated this thread.
1. We do not need to merge those changes upstream, that is too much work and is not worth it. Just update the develop branch with the updated library and make sure everything works.
2. I don't know if the _way_ that I updated the library in that branch was the right way to do it, I did it in haste and it may have been a hacky way to do it, so make sure you are doing it the right way! :grin:
